### PR TITLE
[PBW-4457] CC button always showing in Safari

### PR DESCRIPTION
**Pull request by brianfwong**
Adds a condition to check if the textTrack for the video element has a
src associated with it.

### CODE REVIEW

**Comment by rchalooyala** [comment]
Nice

File: src/main/js/main_html5.js [hunk 1]
```diff
@@ -681,7 +681,7 @@ require("../../../html5-common/js/utils/environment.js");
       if (_video.textTracks && _video.textTracks.length > 0) {
         var languages = [];
         for (var i = 0; i < _video.textTracks.length; i++) {
-          if (_video.textTracks[i].kind === "captions") {
+          if (_video.textTracks[i].kind === "captions" && !_video.textTracks[i].src === "undefined") {
```
Comment (mityaha) [inline src/main/js/main_html5.js]: This doesn't look to me as a correct check for undefined.  It should be either _video.textTracks[i].src != undefined or typeof _video.textTracks[i].src != "undefined".  I actually tested and it doesn't work right.

**Comment by mityaha** [comment]
Check for undefined source wasn't working correctly, it should be either _video.textTracks[i].src != undefined or typeof _video.textTracks[i].src != "undefined".

**Comment by dchauOoyala** [comment]
@mityaha, agreed. In addition, let's use `!==` rather than `!=` to potentially avoid any implicit conversion / coercion that JavaScript may do.

**Pull request merged**